resources/views/account/accept/create.blade.php

```diff
@@ -130,6 +139,24 @@ function resizeCanvas() {
                 $('#signature_output').val(signaturePad.toDataURL());
             }
         });
+        $(document).ready(function(){
+            // Initially hide the div
+            $('#declined_msg').hide();
+            $('#declined_msg_label').hide();
+
+
+            $('input[id="declined"]').change(function(){
+
+                if($(this).is(':checked')){
+                    $('#declined_msg_label').show();
+                    $('#declined_msg').show();
+                }
+                else {
+                    $('#declined_msg_label').hide();
+                    $('#declined_msg').hide();
+                }
+            });
+        });
```

Comment:
This is annoying and not because of this PR but if the "Require Signature" setting is not enabled then the page breaks with a `Uncaught TypeError: Cannot read properties of null (reading 'querySelector')` and this javascript doesn't run. That means the "Add details to why you cant accept this (Optional)" is shown automatically and is still shown when `I accept` is selected...

Reply:
I placed this code on top of the stack and now it does not default to show.